samples/Simple/main.cpp

```diff
@@ -102,7 +104,7 @@ int main(int /*argc*/, char* /*argv*/[])
         NovelRT::Transform(NovelRT::Maths::GeoVector2F(1920 / 2, 1080 / 2), 2, NovelRT::Maths::GeoVector2F(762, 881));
 
     novelChanRect =
-        runner.getRenderer()->createImageRect(novelChanTransform, 3, (imagesDirPath / "novel-chan.png").string(),
+        runner.getRenderer()->createImageRect(novelChanTransform, 0, (imagesDirPath / "novel-chan.png").string(),
```

Comment:
spooky constant: what does 3 and 0 mean?

Reply:
It's the layer value. This sample isn't sticking around much longer anyways. I'm deleting the entire legacy pipeline once everything in ECS is working and in for the MVP.

Reply:
File has been deleted. Resolving.